### DISCUSSION TITLE
I cant quit the cli

### DISCUSSION THREAD

**Discussion by ghost** [traefik/yaegi#1511] [Q&A] [answered]
### The following program `sample.go` triggers an unexpected result

```go
exit/quit in yaegi
```

### Expected result

```console
$ It quits yaegi
```

### Got

```console
$ Im stuck in yaegi for eternity like vim
```

### Yaegi Version

latest

**Comment by KiranSatyaRaj** [comment, marked as answer]
This might help [quit yaegi](https://github.com/traefik/yaegi/discussions/1424)

**Comment by mvertes** [comment]
Yes, `Ctrl-D` will quit, as in many Unix interactive programs, but it should be good also to provide easy commands at least to get help.
